fulllogging.properties

```diff
@@ -0,0 +1,17 @@
+# Global logging properties.
```

Comment:
Being able to _control_ jmDNS logging is a good thing.  And once we have that, turning it off by default (with the option to turn back on) is also a good thing.  We're getting 5k+ lines of irrelevant mDNS logs at the top of every Travis check, making it hard to see the content that matters.  A local user is producing almost 20MB/hr of logs.  

I'm having to serially debug this via Travis because I don't get any mDNS output on my local setup.  If you can think of any way to force some, that would simplify things.

Right now, I'm not convinced that jmDNS is actually obeying the usual Java logging controls.  I haven't been able to get it to modify it's behavior using the usual techniques.  I know that the .jar we have is not the same as the source code, so when they built our current version (in 2011) something odd happened.